forgerock-auth/src/main/java/org/forgerock/android/auth/callback/AbstractCallback.java

```diff
@@ -122,4 +119,11 @@ private JSONObject getInput(JSONObject content, int index) throws JSONException
 
     }
 
+    public String getContent() {
+        return this.content;
+    }
+
+    public int get_id() {
```

Comment:
let me check andy on this, this got generated automatically when i delombak this class,  so that kotlin classes can talk to this class, snake case are mostly used in constants in Java

Reply:
@cerebrl Yes, we try to align the exact naming with the callback
`
   "callbacks": [
        {
            "type": "NameCallback",
            "output": [
                {
                    "name": "prompt",
                    "value": "User Name"
                }
            ],
            "input": [
                {
                    "name": "IDToken1",
                    "value": ""
                }
            ],
            "_id": 0
        },
        {
            "type": "PasswordCallback",
            "output": [
                {
                    "name": "prompt",
                    "value": "Password"
                }
            ],
            "input": [
                {
                    "name": "IDToken2",
                    "value": ""
                }
            ],
            "_id": 1
        }
    ]
}
`
SDK may not necessarily expose this information, however, it may be helpful for developers to know the index of the callback.